packages/shortest/index.d.ts

```diff
@@ -26,13 +26,14 @@ declare module "@antiwork/shortest" {
     expect(description: string): TestChain;
     expect(
       description: string,
-      fn?: (context: TestContextProps) => Promise<void>,
+      fn?: (context: TestContextProps) => Promise<void>
```

Comment:
My editor is doing this even though I have eslint/prettier autosave; any ideas? Doesn't do the same in other files. Also, shouldn't auto fix auto-correct these?

Reply:
looks like the , character is consistently removed in this file. I think for this we need to add a new rule in eslint.